modules/installation-osp-setting-worker-affinity.adoc

```diff
@@ -0,0 +1,123 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_openstack/installing-openstack-installer.adoc
+// * installing/installing_openstack/installing-openstack-installer-custom.adoc
+// * installing/installing_openstack/installing-openstack-installer-kuryr.adoc
+// * installing/installing_openstack/installing-openstack-installer-restricted.adoc
+// * installing/installing_openstack/installing-openstack-user.adoc
+// * installing/installing_openstack/installing-openstack-user-kuryr.adoc
+
+
+[id="installation-osp-setting-worker-affinity_{context}"]
+= Setting compute machine affinity
+
+Optionally, you can set the affinity policy for compute machines during installation.
```

Comment:
I don't know much about OCP on OSP, so I admit that I am not the target audience.  That said, I'm unclear what business outcome I can achieve by setting affinity (or anti-affinity).

In the foll TIP, the link seems like a tip, but the other two pieces of information seem like they could be general text.  Clearly up to you.

On line 25, regarding mention of machine sets, is one mechanism better in some circumstances? How does a customer choose?  (Reminder: I'm not the target audience, so if you believe customers can decide, that's OK.)